### PR TITLE
Add variables examples

### PR DESCRIPTION
**Pull request by zdne**
Also start using the term “Refract” instead of “DOM”.

### CODE REVIEW

File: namespaces/mson-namespace.md [hunk 1]
```diff
@@ -489,6 +497,65 @@ Description is here! Properties to follow.
...
+```
+
+#### MSON Refract
+
+```json
+["object", {}, {}, [
+    ["array",  {"name": "p"}, {}, [
+      [
+        ["generic", {}, {}, "T"]
+      ]
```
Comment (smizell) [inline namespaces/mson-namespace.md]: Extra brackets here?

File: namespaces/mson-namespace.md [hunk 2]
```diff
@@ -70,15 +72,15 @@ Note: Not every MSON _Base Type_ is presented in Refract primitive types and vic
 |     object     |  Object Element  |     object     |   Object Type  |
 |      null      |   Null Element   |        -       |        -       |
 
-# MSON DOM Elements
+# MSON Refract Elements
```
Comment (smizell) [inline namespaces/mson-namespace.md:75]: Do you need "Refract" here?

Comment (smizell) [inline namespaces/mson-namespace.md:75]: Ah, misread. Looks ok.

File: namespaces/mson-namespace.md [hunk 3]
```diff
@@ -489,6 +497,65 @@ Description is here! Properties to follow.
...
+- p: *42*
+```
+
+#### MSON Refract
+
+```json
+["object", {}, {}, [
+    ["string", {"name": "p"}, {"typeAttributes": ["sample"]}, 42]
+  ]]
+```
```
Comment (smizell) [inline namespaces/mson-namespace.md:514]: Is spacing correct here? Four spaces for line 512 and two spaces for 513?

File: namespaces/mson-namespace.md [hunk 4]
```diff
@@ -489,6 +497,65 @@ Description is here! Properties to follow.
...
+
+#### MSON Refract
+
+```json
+["object", {}, {}, [
+    ["string",
+      {"name": ["Relation", {}, {"variable": true}, "rel" ] },
+      {},
+      null]
+  ]]
```
Comment (smizell) [inline namespaces/mson-namespace.md]: Same as https://github.com/refractproject/refract-spec/pull/25/files#r31133039

**Comment by smizell** [comment]
@zdne A few small things. Nothing major.

File: namespaces/mson-namespace.md (hunk 1 above)
Comment (zdne) [inline namespaces/mson-namespace.md]: I hate writing  JSONs

**Comment by zdne** [comment]
@smizell updated JSONs

**Comment by smizell** [comment]
Looks good to me. Can you squash commits?

**Comment by zdne** [comment]
@smizell the two fixes commits were squashed

**Pull request merged**